diff --git a/node_modules/react-native/scripts/react-native-xcode.sh b/node_modules/react-native/scripts/react-native-xcode.sh
index 6e80f84..7d9f82a 100755
--- a/node_modules/react-native/scripts/react-native-xcode.sh
+++ b/node_modules/react-native/scripts/react-native-xcode.sh
@@ -119,7 +119,7 @@ BUNDLE_FILE="$DEST/main.jsbundle"
   --entry-file "$ENTRY_FILE" \
   --platform ios \
   --dev $DEV \
-  --reset-cache \
+  --reset-cache --sourcemap-output ios-release.bundle.map \
   --bundle-output "$BUNDLE_FILE" \
   --assets-dest "$DEST" \
   $EXTRA_PACKAGER_ARGS
